.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: build
```

Comment:
It is very minor but we can use more specific name for the action, I think. What about `PR Builder`? :) @leszko

Reply:
The thing is that AFAIK It's not just for PRs, it also builds master. So, it's quite generic, then the name is also generic `build` ;)

Reply:
I see 👍 :)